pkg/messagebroker/messagebroker.go

```diff
@@ -28,6 +28,9 @@ type Admin interface {
 type Producer interface {
 	// SendMessage sends a message on the topic
 	SendMessage(context.Context, SendMessageToTopicRequest) (*SendMessageToTopicResponse, error)
+
+	// Shutdown closes the producer
+	Shutdown(context.Context)
```

Comment:
to keep interfaces similar, should we name this Close()

Reply:
Had to name it `Shutdown`  since `Broker` already had a `Close` interface for `Consumer`